kf_lib_data_ingest/target_api_plugins/kids_first_dataservice.py

```diff
@@ -278,7 +278,9 @@ def build_entity(record, key, get_target_id_from_record):
         return {
             "kf_id": get_target_id_from_record(Biospecimen, record),
             "participant_id": get_target_id_from_record(Participant, record),
-            "external_sample_id": record.get(CONCEPT.BIOSPECIMEN_GROUP.ID),
+            "external_sample_id": record.get(
```

Comment:
need to also do this for build_key above, otherwise the code will never get here

Reply:
made some changes, is this what you meant?

Reply:
yup!